packages/react-ui-core/src/Photo/BackgroundPhoto.js

```diff
@@ -0,0 +1,35 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import themed from 'react-themed'
+
+@themed(['BackgroundPhoto'], {
+  pure: true,
+})
+
+export default class Photo extends PureComponent {
+  static propTypes = {
+    url: PropTypes.string,
+    theme: PropTypes.object,
+    className: PropTypes.string,
+  }
+
+  static defaultProps = {
+    theme: {},
+  }
+
+  render() {
+    const { theme, className, url, alt, ...rest } = this.props
+
+    return (
+      <div
+        style={{ backgroundImage: `url(${url})` }}
```

Comment:
If we want to allow for a generic fallback, we can use something like

`
    style={{ backgroundImage: `url(${url})${fallbackUrl ? `,url(${fallbackUrl})` : ''}` }}
`